lib/index.js

```diff
@@ -24,19 +24,30 @@ class ElectronDownloader {
   }
 
   get baseUrl () {
-    return process.env.NPM_CONFIG_ELECTRON_MIRROR ||
```

Comment:
This change makes this a major version bump, if we don't reverse the order of these two environment variables it would only be a minor bump.  Can we try keep this change non-breaking?

Reply:
I'd imagine this applies to `middleUrl` and `urlSuffix` too?